Doc/faq/windows.rst

```diff
@@ -140,7 +140,7 @@ offender.
 How do I make an executable from a Python script?
 -------------------------------------------------
 
-See `cx_Freeze <https://anthony-tuininga.github.io/cx_Freeze/>`_ for a distutils extension
```

Comment:
Related to this question/answer, it looks like py2exe now supports Python 3, so this answer can be edited further.

Reply:
Hmm, it's marked as beta, with no changed since 2014. Does that count as supporting Python 3?

Reply:
https://pypi.org/project/py2exe/

Reply:
@akuchling Could you take a look at my comments when you have some spare time? :)

Reply:
Nevermind, on Oct 24 a new version came out. I'll update the PR as suggested :)